gef-ui/templates/_macros/change-something-else-details.njk

```diff
@@ -0,0 +1,27 @@
+{% from "govuk/components/details/macro.njk" import govukDetails %}
+
+{% macro render() %}
+
+{% set AMENDMENT_FORM_EMAIL = "stb.pim@ukexportfinance.gov.uk" %}
```

Comment:
Sorry @BethThomas141 I mean a constant e.g `constants/emails.ts`, passed to the template via the controller, rather than a nunjucks variable

Reply:
Gotcha, have now done this!